build-gradle/action.yml

```diff
@@ -42,6 +43,10 @@ inputs:
     description: URL for Repox
     required: false
     default: https://repox.jfrog.io
+  primary-sonar-platform:
```

Comment:
Will it be better to rename this parameter to `sonar-platform`? For me the `primary-` prefix can mean that there is a `secondary-` parameter.. It's not a strong opinion, just a matter of making things clearer.